server/checks/Specification.js

```diff
@@ -0,0 +1,110 @@
+import Check from './Check'
+import { logger } from '../../common/debug'
+import * as EVENTS from '../model/GithubEvents'
+
+const CHECK_TYPE = 'specification'
+const ACTIONS = ['opened', 'edited', 'reopened', 'synchronize']
+
+const DEFAULT_REQUIRED_LENGTH = 8
+const ISSUE_PATTERN = /^[^#\s]*#\d+$/
```

Comment:
Is this maybe too loose? It would also match `bar/foo.baz&i.am.totally.kidding.you#0`. We could at least require the part before the hash to be of the form `<owner>/<repo>`, ie `^(?:\w+\/\w+)?#\d+$`.